### PR TITLE
Bump up DocStringExtensions and fix some docstrings

### PR DESCRIPTION
**Pull request by navidcy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ and [Navid C. Constantinou][] (@navidcy).
 
 The code is citable via [zenodo](https://zenodo.org). Please cite as:
 
-> Navid C. Constantinou & Gregory L. Wagner and co-contributors. (2021). FourierFlows/FourierFlows.jl: FourierFlows v0.9.0 (Version v0.9.0). Zenodo.  [http://doi.org/10.5281/zenodo.1161724](http://doi.org/10.5281/zenodo.1161724)
+> Gregory L. Wagner, Navid C. Constantinou, and contributors. (2022). FourierFlows/FourierFlows.jl: FourierFlows v0.9.1 (Version v0.9.1). Zenodo.  [http://doi.org/10.5281/zenodo.1161724](http://doi.org/10.5281/zenodo.1161724)
```
Comment (glwagner) [inline README.md:105]: Why this change? I'm ok if you keep the original <3

Comment (navidcy) [reply README.md:105]: I thought we do it based on ideas.
